libraries/plugins/rc/include/steem/plugins/rc/rc_curve.hpp

```diff
@@ -0,0 +1,38 @@
+
+#pragma once
+#include <steem/plugins/rc/rc_utility.hpp>
+
+#include <steem/protocol/types.hpp>
+
+#include <fc/time.hpp>
+
+namespace steem { namespace plugins { namespace rc {
+
+struct rc_curve_gen_params
+{
+   uint16_t                inelasticity_threshold_num    = 1;
```

Comment:
why not one of `size_t`, `uint64_t`, or `unsigned int`?

Reply:
Let's make these uint64_t and then bound them to uint16_t in `validate`. That way we can remove casting in `generate_rc_curve_params`.